workflows/diagnostics/fv3net/diagnostics/offline/_select.py

```diff
@@ -1,16 +1,22 @@
 import numpy as np
 from typing import Sequence
 import xarray as xr
+import cftime
 
 from vcm.select import meridional_ring
 import vcm
 
 
-def nearest_time(select_time: str, times: Sequence[str]):
-    select_datetime = vcm.parse_datetime_from_str(select_time)
-    datetimes = np.vectorize(vcm.parse_datetime_from_str)(times)
-    closest_datetime = min(datetimes, key=lambda d: abs(d - select_datetime))
-    return vcm.encode_time(closest_datetime)
+def nearest_time_batch_index(
+    time: cftime.DatetimeJulian, time_batches: Sequence[Sequence[cftime.DatetimeJulian]]
+):
```

Comment:
Can you add a docstring and output type hint? Don't you need two integers - the batch index, and the time index within the batch?

Update: I see from its use in offline/compute.py that you don't, adding a description of the output here would clarify.

Reply:
These commits were actually already merged to master on another PR. They are no longer in the PR preview now that I've merged master in. Let me know if you feel strongly, I could make another PR to add a docstring.

Reply:
I don't touch this code often, I just found it confusing and would have saved a minute or two of investigation if there were a docstring. I'll leave it up to you.